.circle/configure-rabbitmq.sh

```diff
@@ -2,7 +2,7 @@
 
 # Enable remote guest access
 CONFIG=$(cat <<EHD
-[{rabbit, [{disk_free_limit, 10}, {loopback_users, []}]}].
+[{rabbit, [{disk_free_limit, 10}, {loopback_users, []}, {tcp_listeners, [{"0.0.0.0", 5672}]}]}].
```

Comment:
Same is about `mongo` and `postgresql`.
To keep in mind when thinking about just added tests and packages testing (to avoid conflicts).